scheduler/scheduler-api/src/main/resources/org/ow2/proactive/scheduler/common/xml/schemas/jobdescriptor/dev/schedulerjob.rnc

```diff
@@ -30,13 +30,19 @@ jobName = ## Identification of this job
 variables = ## Definition of variables which can be reused throughout this descriptor
     element variables { variable+ }
 variable = ## Definition of one variable, the variable can be reused (even in another following variable definition) by using the syntax ${name_of_variable}
-    element variable { variableName, variableValue, variableModel?}
+    element variable { variableName, variableValue, variableModel?, variableDescription?, variableGroup?, variableAdvanced? }
```

Comment:
I forgot to mention it, but could you please add these changes in schemas/globalvariables

(you will need to create a new folder 3.13, and also change the schema version in the config/scheduler/global_variables.xml file)